src/MusicStore/Startup.cs

```diff
@@ -36,8 +36,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.Configure<AppSettings>(Configuration.GetSubKey("AppSettings"));
 
-            //Sql client not available on mono
-            var useInMemoryStore = _runtimeEnvironment.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase);
+            // Use the in memory store when running on mono or on the Nano server
+            var useInMemoryStore = _runtimeEnvironment.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Can you move the check condition for Mono in the Platform class ? You can replace this usage through out the project making it consistent